Source/WebKit/UIProcess/Cocoa/WKWebViewContentProviderRegistry.mm

```diff
@@ -56,7 +57,7 @@ - (instancetype)initWithConfiguration:(WKWebViewConfiguration *)configuration
 #endif
 
 #if USE(SYSTEM_PREVIEW) && !HAVE(UIKIT_WEBKIT_INTERNALS)
-    if (configuration._systemPreviewEnabled && !configuration.preferences._modelDocumentEnabled) {
+    if (![_WKSystemPreferences isCaptivePortalModeEnabled] && configuration._systemPreviewEnabled && !configuration.preferences._modelDocumentEnabled) {
```

Comment:
This looks suspicious. Even though lock down mode may be active at global/OS level, it may be disabled for the current site / view. Therefore, making decisions purely on the global flag is likely wrong

Reply:
We don't do checks in the UI process for other disabled content types, so just removing this for now.